docs/user/README.md

```diff
@@ -141,3 +141,5 @@ Installation:
 ```
 
 * You can now invoke the command palette and type LSP enable to locally/globally enable the rust-analyzer LSP (type LSP enable, then choose either locally or globally, then select rust-analyzer)
+
+* Note that `ra_lsp_server` binary must be in `$PATH` for this to work. If it's not the case, you can specify full path to the binary, which is typically `.cargo/bin/ra_lsp_server`.
```

Comment:
This is not enough. Having found `ra_lsp_server` by the full path it then fails running `cargo metadata`.

Reply:
Hm, yeah...

I guess this can be fixed on our side, filed https://github.com/rust-analyzer/rust-analyzer/issues/1813